File: SuperMessenger/SuperMessenger/Models/SentFileModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SuperMessenger.Models
{
    public class SentFileModel
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public Guid ContentId { get; set; }
        public DateTime SendDate { get; set; }

        public Guid GroupId { get; set; }
        public SimpleUserModel User { get; set; }
    }
}

File: SuperMessenger/SuperMessenger/react-client/src/containers/Models/SentFileModel.js
export default class SentFileModel { 
  constructor(id, name, contentId, sendDate, groupId, user, isConfirmed) {
    this.id = id;
    this.name = name;
    this.contentId = contentId;
    this.sendDate = sendDate;
    this.groupId = groupId;
    this.user = user;
    this.isConfirmed = isConfirmed;
  }
}